docs/developing/developing_secrets.md

```diff
@@ -0,0 +1,47 @@
+---
+
+copyright:
```

Comment:
Do we want to add `title` and `description` for these pages to the front matter?

Reply:
I copied this from developingstart_examples.md. It doesnt have those either.